src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private final String supplyString = "supply";
+    private final String buyString = "buy";
+    private final String resultString = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            String resultat = createReport(fromFileName);
+            writeToFile(toFileName,resultat);
+        } catch (IOException e) {
+            throw new RuntimeException(e.getMessage());
+        }
+    }
+
+    public String[] readFromFile(String fromFileName) throws IOException {
+        String[] lines = String.join(",", Files
+        .readAllLines(Paths.get(fromFileName), StandardCharsets.UTF_8)).split(",");
+        return lines;
```

Comment:
` String.join(",", ... ).split(",");` redudant
```suggestion
        return Files.readAllLines(Paths.get(fromFileName), StandardCharsets.UTF_8));
```

Reply:
This action is not redundant, because as you want to do, it will return a List, but I need an array